XenModel/Messages.resx

```diff
@@ -11496,6 +11572,14 @@ To learn more about the [XenServer] Performance Reporting capabilities feature o
 
 To learn more about the [XenServer] Role Based Access Control feature or to start a [XenServer] trial, click the button below.</value>
   </data>
+  <data name="UPSELL_BLURB_VMSS" xml:space="preserve">
+    <value>Upgrade your [XenServer] license to enable VM Snapshot Schedule. VM Snapshot Schedule allows you to create automatic snapshots for your critical VMs.</value>

```

Comment:
No, sorry, I don't think that works.

The feature name is "Scheduled Snapshots" (actually, that's better than the "VM Scheduled Snapshots" I used above).
A single VMSS is a "Snapshot Schedule".
A "scheduled snapshot" is a single snapshot, created according to the schedule.

I have made the same comment on xapi-project/xen-api#2577